### PR TITLE
Add validation as a new pattern

### PR DESCRIPTION
**Pull request by quis**
Based on the code used in the SSP.

image

### CODE REVIEW

File: pages_builder/pages/forms/validation.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      </div>
+      <div class="validation-wrapper">
+        <div class="question">
+          <label for="example-textbox" class="question-heading">
+            What was the name of your first pet?
+          </label>
+          <p class="validation-message">
+            This question requires an answer
+          </p>
+          <input class="text-box" id="example-textbox" type="text" />
```
Comment (tombye) [inline pages_builder/pages/forms/validation.yml]: Following advice from Léonie, we should add `aria-describedby` to the `<input>` linking it to the `validation-message` paragraph.

**Comment by tombye** [comment]
Think this needs linking to from the index page.

**Comment by quis** [comment]
Link from forms/index added in e5ead43.

File: pages_builder/pages/forms/validation.yml (hunk 1 above)
Comment (quis) [inline pages_builder/pages/forms/validation.yml]: Added in 935d7ba.

**Comment by tombye** [comment]
One thing I forgot to mention: can we wrap it in a form so it's clear the markup that should be used?

**Comment by quis** [comment]
@tombye  d447cb2, good idea

**Pull request merged**